test/internal/kubernetes/utils_create.go

```diff
@@ -429,6 +472,12 @@ func setupCNSDaemonset(
 		return appsv1.DaemonSet{}, cnsDetails{}, errors.Wrap(err, "failed to parse cns daemonset")
 	}
 
+	if cnsScenarioDetails.installIPMasqAgent {
```

Comment:
We should add a log message about `InstallIPMasqAgent`. Either inside the function or before the call.

Reply:
Done